samza-test/src/main/python/configs/downloads.json

```diff
@@ -1,5 +1,5 @@
 {
-  "url_kafka": "http://www.us.apache.org/dist/kafka/0.8.2.0/kafka_2.9.2-0.8.2.0.tgz",
-  "url_zookeeper": "http://archive.apache.org/dist/zookeeper/zookeeper-3.4.3/zookeeper-3.4.3.tar.gz",
+  "url_kafka": "http://archive.apache.org/dist/kafka/0.10.0.1/kafka_2.10-0.10.0.1.tgz",
```

Comment:
Just realized one thing after merging PR #28 : can we make the scala version a variable to be expanded here? e.g. defining a ${scalaVersion}, and use:
"url_kafka": "http://archive.apache.org/dist/kafka/0.10.0.1/kafka_${scalaVersion}-0.10.0.1.tgz" instead?

The same comments to many following changes in samza-test module.